graylog2-server/src/main/java/org/graylog2/shared/users/ChangeUserRequest.java

```diff
@@ -75,7 +79,8 @@ public static ChangeUserRequest create(@JsonProperty("email") @Nullable @Email S
                                            @JsonProperty("timezone") @Nullable String timezone,
                                            @JsonProperty("startpage") @Nullable @Valid Startpage startpage,
                                            @JsonProperty("session_timeout_ms") @Nullable @Min(1) Long sessionTimeoutMs,
-                                           @JsonProperty("roles") @Nullable List<String> roles) {
-        return new AutoValue_ChangeUserRequest(email, firstName, lastName, permissions, timezone, startpage, sessionTimeoutMs, roles);
+                                           @JsonProperty("roles") @Nullable List<String> roles,
+                                           @JsonProperty("service_account") @Nullable Boolean isServiceAccount) {
+        return new AutoValue_ChangeUserRequest(email, firstName, lastName, permissions, timezone, startpage, sessionTimeoutMs, roles, isServiceAccount);
```

Comment:
Let's handle a `null` value for `isServiceAccount` here. (see comment above)

Reply:
fixed